isotope-intensity-estimation.Rmd

```diff
@@ -210,70 +222,81 @@ of one or more neutrons, the difference in mass between them isn't exactly
 integer and each elemental isotope will contribute in a different 
 way producing different overall mass differences. 
 
-In a compound with $n_X$ atoms of element $X$ it is possible that 
-${n_X}_1$, ${n_X}_2$, ..., ${n_X}_q$ (${n_X}_1+{n_X}_2+ ... +{n_X}_q = n_X$) 
-atoms of $X$ have isotope form $1, 2, ... q$. Each configuration 
-$C_X = ({n_X}_1, {n_X}_2, ..., {n_X}_q)$ will result in a different shift in 
-mass $md_{C_X}$. 
-The probability $P(C_X)$ of observing such a configuration con be computed using 
-the multinomial distribution.
-The overall mass difference $md$ with respect to the monoisotopic mass is 
-determined by the configurations of each element $X, Y,... Z$ in the molecule 
-i.e. by $C = (C_X, C_Y... C_Z)$ and is given by the sum 
-$md = md_{C_X}+ md_{C_Y} ... +md_{C_Z}$. Finally the probability of 
-observing the overall configuration $C$ (and the corresponding mass difference 
-$md$) is $P(C) = P(md) = P(C_X)* P(C_Y)...*P(C_Z)$ i.e. the product of the 
-probability of observing the configurations related to each element.
-
-There is a very high number $N_{C_X} = \binom{q + n_X -1}{n_X}$ of possible 
-configurations $C_X$ for the atoms of $X$ and a even greater 
-number $N_{C_X}* N_{C_Y}...*N_{C_Z}$ of overall configurations 
-$C=(C_X, C_Y..., C_Z)$. Here we keep only those that are significant for our 
-purpose.
-
-We define family of configurations as the sets of configurations that share the 
-same number of heavier isotopes for each element and therefore the same mass 
-difference with respect to the monoisotopic mass.
-For now we considered the following families of configurations related the 
-isotopes of $C, N, O, S, Cl$
-
-$((n_C-1,1), (n_N, 0), (n_O, 0, 0), (n_S, 0, 0), (n_{Cl},0))$ corresponding to 
-the "C13" case
-
-$((n_C-2,2), (n_N, 0), (n_O, 0, 0), (n_S, 0, 0), (n_{Cl},0))$ corresponding to 
-the "2C13" case
-
-$((n_C, 0), (n_N, 0), (n_O-1, 1, 0), (n_S, 0, 0), (n_{Cl},0))$ corresponding to 
-the "O17" case
-
-$((n_C, 0), (n_N, 0), (n_O-1, 0, 1), (n_S, 0, 0), (n_{Cl},0))$ corresponding to 
-the "O18" case
-
-...
-et cetera.
-
-For example ((5,1), (4, 0), (3, 0, 0), (2, 0, 0), (1,0)) and 
-((1,1), (2, 0), (3, 0, 0), (4, 0, 0), (5,0)) are both in the "C13" family and 
-correspond to the md: mass of C13 - mass of C12
-
-## Selecting "significant" families of configurations
-
-Here we try to determine which families of configurations (or equivalently which 
-mass differences with respect to the monoisotopic mass) to keep.
-
-A first idea could be to compute the isotope pattern for each element in HMDB  
-and store the configurations (more precisely, the families associated that 
-configuration) that are related to isotope pattern peaks with intensity higher 
-than a certain threshold (whose value we have to decide). Then we could decide 
-to keep the most frequent ones (and we would have to decide a certain cutoff).
-
-```{r}
+In a compound with $n$ atoms of element $X$ ($n_X$) each of the atoms can be in
```

Comment:
@andreavicini , please check if this is still correct/compliant with your definition.

Reply:
Thanks